README.md

```diff
@@ -45,17 +45,17 @@ Example Applications
 --------------------
 
 - [`BigQueryExample`](./gcloud-java-examples/src/main/java/com/google/gcloud/examples/bigquery/BigQueryExample.java) - A simple command line interface providing some of Cloud BigQuery's functionality
-  - Read more about using this application on the [`gcloud-java-examples` docs page](http://googlecloudplatform.github.io/gcloud-java/apidocs/?com/google/gcloud/examples/BigQueryExample.html).
+  - Read more about using this application on the [`gcloud-java-examples` docs page](http://googlecloudplatform.github.io/gcloud-java/apidocs/?com/google/gcloud/examples/bigquery/BigQueryExample.html).
```

Comment:
NIT: Would it make sense to say that "Read more ... on the [`gcloud-java-examples-bigquery` docs page]" and make the text of the links for (datastore, resource manager and storage specific as well? It feels weird that the page shows me 4 links which are all called "`gcloud-java-examples` docs page" but they take me to different places.